packages/runtime/src/components/createComponent.ts

```diff
@@ -159,6 +159,11 @@ export function createComponent({
       })
     }
   })
+  Object.entries(apis)
+    .filter(([_, api]) => api.triggerActions !== undefined)
```

Comment:
Why/when would `triggerActions` be `undefined`? To avoid the optional chaining on line 165, you could do something like this perhaps. Also I suppose there's no reason to use `Object.entries` if you only use the values:
```
Object.values(apis)
	.filter((api): api is RequireFields<ContextApi, 'triggerActions'> => api.triggerActions !== undefined)
```